editions/tw5.com/tiddlers/filters/trim Operator.tid

```diff
@@ -1,13 +1,16 @@
 caption: trim
 created: 20190613153740241
 modified: 20190613153820282
+op-purpose: returns each item in the list with whitespace, or a given character string, trimmed from the start and/or end
```

Comment:
The docs should clarify that the character string is interpreted as a regular expression, and not as plain text.

Reply:
It's not interpreted as a regular expression; see https://github.com/Jermolene/TiddlyWiki5/pull/4811#issuecomment-683203044 for details.